pkg/provider/aws/action/kind/kind.go

```diff
@@ -116,55 +113,106 @@ func (r *kindRequest) createHost() error {
 				awsConstants.CONFIG_AWS_NATIVE_REGION: *r.allocationData.Region}),
 		DeployFunc: r.deploy,
 	}
-	sr, _ := manager.UpStack(cs)
-	return manageResults(sr, r.prefix)
```

Comment:
Why did you remove this?....if beyond write those results and the price which was missing previously...just add that struct there (KindResutlsMetadata) and return it from the manageResults function no? ...all code for getting each results is already there no?